frontend/ui/src/pages/Channels/ConnectedChannelsList/ChannelsList.tsx

```diff
@@ -74,6 +77,16 @@ const ChannelsList = (props: ChannelsListProps) => {
     }
   };
 
+  const channelSorter = (channelA: Channel, channelB: Channel) => {
+    if (channelA.metadata.name.toLowerCase() < channelB.metadata.name.toLowerCase()) {
+      return -1;
+    }
+    if (channelA.metadata.name.toLowerCase() > channelB.metadata.name.toLowerCase()) {
```

Comment:
same happened here :) this function is removed and the sorting is happening inline with localeCompare()